cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,145 @@
+        output_logits = {output: classifier(cls_output) for output, classifier in self.classifiers.items()}
+        return output_logits
+    
+class MultiOutputDataset(Dataset):
+    def __init__(self, encodings):
+        self.encodings = encodings
+        # self.labels = labels
+
+    def __getitem__(self, idx):
+        item = {key: val[idx].clone().detach() for key, val in self.encodings.items()}
+        # label_item = {output: self.labels[output][idx] for output in outputs}
+        return item
```

Ruff

Unnecessary assignment to `item` before `return` statement

Remove unnecessary assignment

(RET504)